### PR TITLE
Update Japanese translations

### PR DESCRIPTION
**Pull request by tkusano**
@Hima-Zinn @dolciss @oboenikui @noritada @middlingphys @hibiki @reindex-ot @haoyayoi @vyv03354
Please review

- Add GIPHY related messages
  - "Powered by GIPHY"  is left as is without translation
- Add a11y related messagges
- Update a message in empty search screen

image
image
image

### CODE REVIEW

**Review by Hima-Zinn — COMMENTED** [review]
I don't seem to have any problems with it.
I am looking forward to being able to use GIFs.

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -1832,6 +1854,10 @@ msgstr "ギャラリー"
 msgid "Get Started"
 msgstr "開始"
 
+#: src/components/dialogs/GifSelect.tsx:320
+msgid "GIPHY may collect information about you and your device. You can find out more in their <0>privacy policy</0>."
+msgstr "GIPHYはあなたやあなたのデバイスについての情報を収集するかもしれません。GIPHYの<0>プライバシー・ポリシー</0>を見ることができます。"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:1859]: `を見ることができます`→`で詳細を確認できます`とするのはどうでしょうか。

Comment (tkusano) [reply src/locale/locales/ja/messages.po]: 良さそうですね。反映しました。

**Review by pfrazee — APPROVED** [review]
Thanks so much everyone!

**Pull request merged**